src/middleware/find.js

```diff
@@ -1,18 +1,62 @@
-import React from 'react'
-let { TestUtils } = React.addons
+import React from 'react';
```

Comment:
Why add semicolons here?? May be you can add no-semicolon rule to the eslintrc. So, the style is consistent across the codebase.